src/LRS.js

```diff
@@ -747,7 +747,12 @@ TinCan client library
 
             for (i = 0; i < idProps.length; i += 1) {
                 if (typeof cfg.params[idProps[i]] !== "undefined") {
-                    params[idProps[i]] = cfg.params[idProps[i]].id;
+                    if (typeof cfg.params[idProps[i]] === "string") {
+                        params[idProps[i]] = cfg.params[idProps[i]];
+                    }
+                    else {
+                        params[idProps[i]] = cfg.params[idProps[i]].id;
+                    }
```

Comment:
I'm torn, particularly because of how JS `typeof` works on whether we should be more direct and check explicitly for `TinCan.Verb` or `TinCan.Activity`, but I guess I could be convinced otherwise, particularly for backward compatibility with objects that had an `id` property but weren't those types. Having said that it does perhaps beg the question of whether it makes more sense to do `if (typeof cfg.params[idProps[i]].id !== "undefined") {` rather than checking for string.